components/scream/src/physics/shoc/shoc_functions_f90.cpp

```diff
@@ -2247,7 +2247,9 @@ void update_prognostics_implicit_f(Int shcol, Int nlev, Int nlevi, Int num_trace
 
   view_3d
     tracer_d(temp_3d_d[0]);
-  Kokkos::resize(Kokkos::WithoutInitializing, tracer_d,
+
+  // update_prognostics_implicit() expects 3 extra tracer slots
+  Kokkos::resize(tracer_d,
                  shcol,nlev,ekat::npack<Spack>(num_tracer+3));
```

Comment:
The changes in this file do not affect computations. `Kokkos::WithoutInitializing` is unnecessary.